### DISCUSSION TITLE
Where have you been my whole life?

### DISCUSSION THREAD

**Discussion by nacho-domatix** [jkwill87/mnamer#112] [General]
This is not an issue, but a big thanks. It's an amazing (and needed) software. I tried filebot (which is not opensource neither free right now), tvnamer (only for tvseries) and recently imdb-renamer but this is the more complete and effective tool for cli.

Thank you very much and keep sharing your work :)

**Comment by jkwill87** [comment]
Haha, when I get an email about a GitHub issue it usually follows with a feeling of dread-- oh boy, what did I break this time. Thanks!

**Comment by k8ieone** [comment]
Hello from another person who really appreciates this tool. I've been looking for this for so long and I love it so much!

This definitely goes to my personal "essential tools" list.

**Comment by aztazt** [comment]
Same from here, amazing tool, thank you so much for sharing it with community !

**Comment by cchandler** [comment]
I'll heap some love on too. I just found it in 2025 and it works pretty much exactly the way I think it should 😎. Thanks for building this!
